rinja/src/filters/mod.rs

```diff
@@ -2,6 +2,13 @@
 //!
 //! Contains all the built-in filter functions for use in templates.
 //! You can define your own filters, as well.
+//!
+//! ## Note
+//!
+//! All **result types of any filter function** in this module is **subject to change** at any
+//! point, and is **not indicated by as semver breaking** version bump.
+//! The traits [`AutoEscape`] and [`WriteWritable`] are used by [`rinja_derive`]'s generated code
+//! to work with all compatible types.
```

Comment:
Adding this change is semver breaking in itself, but IMO its still worth it to keep our options open in the future or add more optimizations in the future, e.g. because writing filters is currently not optimized.

Reply:
As long as it doesn't break users code, I'm fine with it.